packages/auth/__tests__/selling-partner-api-auth.spec.ts

```diff
@@ -75,29 +75,37 @@ describe('src/selling-partner-api-auth', () => {
         }
       })
     } catch (error: unknown) {
-      expect(error).toBeInstanceOf(SecurityTokenServiceError)
+      expect(error).toBeInstanceOf(SellingPartnerApiAuthError)
     }
   })
 
-  it('should not fail for missing accessKeyId', () => {
-    expect.assertions(0)
+  it('should fail for missing accessKeyId', () => {
+    expect.assertions(1)
 
-    void new SellingPartnerApiAuth({
-      clientId: 'FAKE_CLIENT_ID',
-      clientSecret: 'FAKE_CLIENT_SECRET',
-      secretAccessKey: 'FAKE_SECRET_ACCESS_KEY',
-      refreshToken: 'FAKE_REFRESH_TOKEN'
-    })
+    try {
+      void new SellingPartnerApiAuth({
+        clientId: 'FAKE_CLIENT_ID',
+        clientSecret: 'FAKE_CLIENT_SECRET',
+        secretAccessKey: 'FAKE_SECRET_ACCESS_KEY',
+        refreshToken: 'FAKE_REFRESH_TOKEN'
+      })
+    } catch (error: unknown) {
+      expect(error).toBeInstanceOf(SellingPartnerApiAuthError)
+    }
```

Comment:
This could also be done like this:

```ts
expect(() => new SellingPartnerApiAuth({
  clientId: 'FAKE_CLIENT_ID',
  clientSecret: 'FAKE_CLIENT_SECRET',
  secretAccessKey: 'FAKE_SECRET_ACCESS_KEY',
  refreshToken: 'FAKE_REFRESH_TOKEN'
})).toThrow(SellingPartnerApiAuthError)
```

Then you won’t have the need for `expect.assertions(1)`.